### PR TITLE
[翻译更新] utilities

### PR DESCRIPTION
**Pull request by liaopeiyuan**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 1]
```diff
@@ -204,18 +201,20 @@ msgstr ""
...
-"gdb thread attaching. After entering ``gdb``, check the stack backtrace "
-"with command ``bt`` (``backtrace``), then find the line of code "
-"triggering the error."
+"**Quickly pinpointing segmentation faults/assertion failures using** ``gdb``: "
+"When Taichi crashes, ``gdb`` will be triggered and attach to the current "
+"thread. You might be prompt to enter sudo password required for gdb thread "
+"attaching. After entering ``gdb``, check the stack backtrace with command "
+"``bt`` (``backtrace``), then find the line of code triggering the error."
 msgstr ""
+"**快速定位存储器区段错误和断言错误** ``gdb``: 在Taichi 崩溃时，``gdb`` 会被响应"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]: We add English text in brackets to help Chinese users connect their knowledge in programming (e.g. `assert(x)`).
```suggestion
"使用 ``gdb`` **快速定位段错误 (segmentation fault) 和断言错误 (assertion failure)**: 在Taichi 崩溃时，``gdb`` 会被响应"
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 2]
```diff
@@ -204,18 +201,20 @@ msgstr ""
...
+"启动并附着到当前的线程上。您可能需要输入和 ``sudo`` 超级用户权限相关的密码来允"
+"许 gdb 附着到当前的线程上。再输入 ``gdb`` 之后， 您可以使用 ``bt`` "
+"（``backtrace``）指令检查堆的回溯，因而定位产生错误的代码的具体行数。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
"（``backtrace``）指令检查堆的回溯，从而定位产生错误的代码的具体行数。"
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 3]
```diff
@@ -125,62 +124,60 @@ msgstr ""
...
+"定性能的机器作为时间基准服务器。 讨论在: https://github.com/taichi-dev/taichi/"
+"issue/948"
 
 #: ../../utilities.rst:58
-#, fuzzy
 msgid ""
 "The suggested workflow for the performance related PR author to run the "
 "regression tests is:"
-msgstr "建议 **the PR author** 运行回归测试的工作流程是："
+msgstr "建议 **the PR author** （Pull Request 的作者）运行回归测试的工作流程是："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
msgstr "建议 **Pull Request 的作者** 运行回归测试的工作流程是："
```

**Review by archibate — COMMENTED** [review]
Thank for updating! LGTM + nits.

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 4]
```diff
@@ -125,62 +124,60 @@ msgstr ""
...
 
 #: ../../utilities.rst:62
 msgid "Run ``git checkout -b your-branch-name``."
-msgstr ""
+msgstr "运行 ``git checkout -b your-branch-name`` （分支名称）."
 
 #: ../../utilities.rst:64
 msgid "Do works on the issue, stage 1."
-msgstr ""
+msgstr "开始尝试解决问题（第一阶段）。"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
msgstr "尝试解决问题（第一阶段）。"
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 5]
```diff
@@ -204,18 +201,20 @@ msgstr ""
...
-"with command ``bt`` (``backtrace``), then find the line of code "
-"triggering the error."
+"**Quickly pinpointing segmentation faults/assertion failures using** ``gdb``: "
+"When Taichi crashes, ``gdb`` will be triggered and attach to the current "
+"thread. You might be prompt to enter sudo password required for gdb thread "
+"attaching. After entering ``gdb``, check the stack backtrace with command "
+"``bt`` (``backtrace``), then find the line of code triggering the error."
 msgstr ""
+"使用 ``gdb`` **快速定位段错误 (segmentation fault) 和断言错误 (assertion failure)**: 在Taichi 崩溃时，``gdb`` 会被响应"
+"启动并附着到当前的线程上。您可能需要输入和 ``sudo`` 超级用户权限相关的密码来允"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
"使用 ``gdb`` **快速定位段错误 (segmentation fault) 和断言错误 (assertion failure)**: 在Taichi 崩溃时，``gdb`` 会被"
"触发并附着到当前的线程上。您可能需要输入和 ``sudo`` 超级用户权限相关的密码来允"
```

**Review by isdanni — APPROVED** [review]
Thanks for the contribution! Just a few nips :-)

btw sorry for the delay..:skull::skull::skull:spilt water on my computer & spent two days fixing it

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 6]
```diff
@@ -226,20 +225,18 @@ msgid "ti.core.print_all_units()"
...
+"The serialization module of taichi allows you to serialize/deserialize objects "
+"into/from binary strings."
 msgstr "Taichi的序列化模块可以允许你将对象序列化/反序列化成二进制字符串。"
 
 #: ../../utilities.rst:112
 msgid ""
-"You can use ``TI_IO`` macros to explicit define fields necessary in "
-"Taichi."
+"You can use ``TI_IO`` macros to explicit define fields necessary in Taichi."
 msgstr "你可以使用 ``TI_IO`` 宏来显式定义太极拳中必要的字段。"
```
Comment (rexwangcc) [inline locales/zh_CN/LC_MESSAGES/utilities.po]: Hmm, let's remove the `拳` and use `Taichi` here 🤣  (It's not what this PR changed, but might be better to fix ASAP...)

**Review by rexwangcc — CHANGES_REQUESTED** [review]
LGTM in general!! Could you help fix the typo that was introduced before as well? Thanks!

**Review by rexwangcc — APPROVED** [review]

**Pull request merged**